pages/home.tsx

```diff
@@ -25,71 +20,89 @@ interface Course {
 
 interface TitlebarImageListProps {
   data: Course[];
-  listIndex: number;
-  length: number;
-  setNext(index: number): void;
 }
 
-const TitlebarImageList: React.FC<TitlebarImageListProps> = ({ data, listIndex, length, setNext }) => {
-  const [transitionDirection, setTransitionDirection] = useState<'left' | 'right' | null>(null);
-
-  useEffect(() => {
-    // Reset transition direction after the transition ends
-    const transitionEndHandler = () => setTransitionDirection(null);
-    const imageList = document.querySelector('.home_imageList') as HTMLElement;
-    imageList.addEventListener('transitionend', transitionEndHandler);
-
-    return () => {
-      imageList.removeEventListener('transitionend', transitionEndHandler);
-    };
-  }, [transitionDirection]);
-
-  const handleSetNext = (direction: 'left' | 'right') => {
-    setTransitionDirection(direction);
-    setNext(direction === 'left' ? listIndex - 1 : listIndex + 1);
-  };
+const TitlebarImageList: React.FC<TitlebarImageListProps> = ({ data }) => {
   return (
-    <div 
-      className={`imageListContainer ${transitionDirection} flexRowCenter`}
-    >
-      {listIndex>0 && <IconButton
-        color="inherit"
-        onClick={() => handleSetNext('left')}
```

Comment:
try setting the arrows the purple primary color we have. If it doesnt look good leave it.